tamr_unify_client/project/attribute_mapping/resource.py

```diff
@@ -64,6 +65,14 @@ def resource_id(self):
         spliced = self.relative_id.split("attributeMappings/")[1]
         return spliced
 
+    @property
+    def delete(self):
+        """:return: :class `requests.response`
+        """
+        delete_url = self.client.origin + self.client.base_path + self.relative_id
+        response = self.client.delete(delete_url).successful()
```

Comment:
So when you make a request with the client, it will handle all the base path stuff for you. All you need to do is pass it the relative_id, and it will concatenate that with the base path information.